ddtrace/profiling/collector/_utils.h

```diff
@@ -0,0 +1,116 @@
+#ifndef _DDTRACE_UTILS_H
+#define _DDTRACE_UTILS_H
+
+#include <Python.h>
+#include <stdlib.h>
+
+#define p_new(type, count) PyMem_RawMalloc(sizeof(type) * (count))
+#define p_delete(mem_p) PyMem_Free(mem_p);
+#define p_alloc_nr(x) (((x) + 16) * 3 / 2)
```

Comment:
A comment here might be useful. These seem like magic numbers

Reply:
Added.